build.sbt

```diff
@@ -1,82 +1,38 @@
-javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint")
-
-lazy val commonSettings = Seq(
-  organization := "Dwolla",
-  homepage := Option(url("https://stash.dwolla.net/projects/OPS/repos/cloudflare-public-hostname-lambda")),
+lazy val buildSettings = Seq(
+  organization := "com.dwolla",
+  name := "cloudformation",
```

Comment:
Yeah. It should definitely be changed to something containing `cloudflare` --- my bad.